testsuite/src/commonMain/kotlin/org/jetbrains/litmuskt/tests/ArrayVolatile.kt

```diff
@@ -18,6 +18,11 @@ object ArrayVolatile {
             var arr = IntArray(2)
         }
     }) {
+        reset {
+            arr[0] = 0
+            arr[1] = 0
+            outcomeReset()
+        }
```

Comment:
Since `reset` block is mostly an implementation detail, I would suggest to follow a convention, and always put it to the end of the test (after `spec` block). So that we first see the logic of the test itself (`thread` blocks), the the expected outcomes (`spec`  block), and the `reset` block as the last thing.